api/docs/installation/localvir.md

```diff
@@ -10,6 +10,12 @@ These are some additional depencies that you will need:
 $ sudo apt-get update
 ```
 
+* Postgres
```

Comment:
Add this before step 3 and also change the heading to Postgres Installation (also make it bold) . :smile:

Reply:
ok sir :+1:

Reply:
> Add this before step 3 
Sir can you please explain what to add?

Reply:
Ok sir will do it  :+1: